site/src/scaffold/scripts/tokens.ts

```diff
@@ -4,3 +4,94 @@ import tokens from '@microsoft/atlas-css/dist/tokens.json';
 import type { AtlasTokens } from '@microsoft/atlas-css/dist/tokens';
 
 export const atlasTokens = tokens as AtlasTokens;
+
+const palette = atlasTokens.palette.tokens;
+
+const listOfBlockedColors = [
+	'blueblack',
+	'bluea',
+	'yellowsand',
+	'yellowhighcontrast',
+	'yellowhighcontrasthover',
+	'purplea',
+	'purpleb'
+];
+
+const formatColorPalette = (palette: Record<string, string>) => {
+	return Object.keys(palette).reduce((acc, key) => {
+		const oldKey = key;
+		key = key.replace('$palette-', '');
+		key = key.replaceAll('-', '');
+		// generic object injection sink is the error. We know where the data is coming from (atlas tokens).
+		//eslint-disable-next-line
+		acc[key] = palette[oldKey];
+		return acc;
+	}, {} as Record<string, string>);
+};
+
+const filterPaletteForAColor = (color: string, colorPalette: Record<string, string>) => {
+	return Object.keys(colorPalette).reduce((acc, key) => {
+		if (key.includes(color) && !key.includes('opacity') && !listOfBlockedColors.includes(key)) {
+			// generic object injection sink is the error. We know where the data is coming from (atlas tokens).
+			//eslint-disable-next-line
+			acc[key] = colorPalette[key];
+		}
+		return acc;
+	}, {} as Record<string, string>);
+};
+
+const renderColorPalleteToHTML = (colors: { [key: string]: string }, containerID: string) => {
+	const container = document.getElementById(`${containerID}`);
+	if (!container) {
+		return console.error(`Could not find container with ID ${containerID}`);
+	}
+	let colorsDivs = '';
+	Object.keys(colors).map(key => {
+		const value = colors[key];
+		return (colorsDivs += `
+		<div style="background-color: ${value}; display: block; width: 120px; height: 70px; color: ${
+			key.includes('120') ||
+			key.includes('110') ||
+			key.includes('100') ||
+			key.includes('90') ||
+			key.includes('80') ||
+			key.includes('70') ||
+			key.includes('60')
+				? 'white'
+				: 'black'
+		}" class="border-radius padding-left-xs padding-top-xs margin-xxs">
+			<h4 class="color-swatch__label padding-bottom-xxs" style="text-transform: capitalize";>${key}</h4>
```

Comment:
We probably don't want to have a bunch of heading elements, as this can clutter the experience for screenreader users. Headings should be used to describe sections of content, and not individual pieces of information.

Instead, I think we should consider using the [`dl` element](https://benmyers.dev/blog/on-the-dl/) for this, since its job is to mark up name-value pairs like this. Here's the markup I'd expect:

```html
<dl id="blueList">
    <div> <!-- apply color inline styles to this wrapper div -->
        <dt>Blue10</dt>
        <dd>#D7EAF8</dd>
    </div>
    <div> <!-- apply color inline styles to this wrapper div -->
        <dt>Blue20</dt>
        <dd>#9CCBEE</dd>
    </div>
    ...
</dl>
```

Alternatively, we just replace these `h4`s with `<p><strong></strong></p>`.